docs/source/openapi.json

```diff
@@ -2270,12 +2270,12 @@
                           "row_idx": 0,
                           "row": {
                             "imageA": {
-                              "src": "https://datasets-server.huggingface.co/assets/huggan/horse2zebra/--/default/train/0/imageA/image.jpg",
+                              "src": "https://datasets-server.huggingface.co/assets/huggan/horse2zebra/main/default/train/0/imageA/image.jpg",
```

Comment:
If we do this, we can no longer parse the URL. The `--` separator ensures we can get the dataset name, canonical or with a namespace.

If we keep it, the URL would be:

```
                              "src": "https://datasets-server.huggingface.co/assets/huggan/horse2zebra/--/main/default/train/0/imageA/image.jpg",
```

We can also remove it if it is useless, but let's be sure it does not break anything